src/main/index.js

```diff
@@ -1653,14 +1792,9 @@ function runApp() {
 
   let resourcesCleanUpDone = false
 
-  app.on('window-all-closed', () => {
+  app.on('window-all-closed', async () => {
```

Comment:
Looks like this change is not necessary.
```suggestion
  app.on('window-all-closed', () => {
```